x-pack/plugins/enterprise_search/jest.sh

```diff
@@ -1,11 +1,19 @@
 #! /bin/bash
 
 # Whether to run Jest on the entire enterprise_search plugin or a specific component/folder
+
 FOLDER="${1:-all}"
```

Comment:
Since this is no longer really accurate, could we rename this to maybe `TARGET`?

```suggestion
# Whether to run Jest on the entire enterprise_search plugin or a specific file/folder

TARGET="${1:-all}"
```

Reply:
6329b4e11658096c8170610fcf486a1b657cb676